packages/forms-web-app/src/controllers/final-comment/check-your-answers.js

```diff
@@ -0,0 +1,49 @@
+const { VIEW } = require('../../lib/views');
+const { storeTextAsDocument } = require('../../services/pdf.service');
+const { documentTypes } = require('@pins/common');
+
+exports.getCheckYourAnswers = async (req, res) => {
+	res.render(VIEW.FINAL_COMMENT.CHECK_YOUR_ANSWERS, {
+		hasComments: req.session?.finalComment?.hasComments,
+		finalComments: req.session?.finalComment
+	});
+};
+
+exports.postCheckYourAnswers = async (req, res) => {
+	const {
+		body: { errors = {}, errorSummary = [] }
+	} = req;
+
+	if (!req.session.finalComment?.finalCommentAsDocument) {
```

Comment:
might be worth commenting that this is dev only stuff that will need to be removed when final comment is set up properly